edx_rest_framework_extensions/config.py

```diff
@@ -2,6 +2,18 @@
 Application configuration constants and code.
 """
 
-NAMESPACE_SWITCH = 'oauth2'
-SWITCH_ENFORCE_JWT_SCOPES = 'enforce_jwt_scopes'
-NAMESPACED_SWITCH_ENFORCE_JWT_SCOPES = '{}.{}'.format(NAMESPACE_SWITCH, SWITCH_ENFORCE_JWT_SCOPES)
+OAUTH_TOGGLE_NAMESPACE = 'oauth2'
+SWITCH_ENFORCE_JWT_SCOPES = '{}.enforce_jwt_scopes'.format(OAUTH_TOGGLE_NAMESPACE)
+
+# .. toggle_name: EDX_DRF_EXTENSIONS: ENABLE_SET_REQUEST_USER_FOR_JWT_COOKIE
```

Comment:
Curious: Are toggle names intended to be prefixed with the repo name?

Reply:
1. Adding a new Django Setting following an existing pattern which namespaces in this way.
2. In general, I like this pattern to help people find settings more easily.  Note that I don't know if our toggle report is yet looking at dependency repos.